src/ServiceManagement/Compute/Commands.ServiceManagement.Test/Resources/VnetconfigWithLocation.netcfg

```diff
@@ -15,7 +15,7 @@
       </LocalNetworkSite>
     </LocalNetworkSites>
     <VirtualNetworkSites>
-      <VirtualNetworkSite name="NewVNet4" Location="Central Us">
+      <VirtualNetworkSite name="NewVNet4" Location="West Us">
```

Comment:
Do you have to change the location to "West US" for your testing?
Currently AzureRT tests are running in "Central US" location, so if you want to change the location, then create a new netcfg file (e.g. VnetconfigWithLocationWestUS.netcfg) rather than change the location of the current netcfg file.